query_execution/QueryContext.proto

```diff
@@ -54,4 +54,6 @@ message QueryContext {
 
   // NOTE(zuyu): For UpdateWorkOrder only.
   repeated UpdateGroup update_groups = 10;
+
+  required uint64 query_id = 11;
```

Comment:
Since we add a new field in `QueryContext` proto, we need to modify `QueryContext::ProtoIsValid` to reflect the change:

```
  return proto.IsInitialized();
```